src/ui/rendering/handlebarsrenderer.js

```diff
@@ -195,7 +195,8 @@ export default class HandlebarsRenderer extends Renderer {
 
     this.registerHelper('runtimeTranslation', function (options) {
       let { phrase, count } = options.hash;
-      return Translator.translate(phrase, options.hash, count);
+      const translation = Translator.translate(phrase, options.hash, count);
+      return self.SafeString(translation);
```

Comment:
I'm a bit concerned about always marking the translations a `SafeString`. My reasons are this:

- This causes us to assume some responsibility for the security of any page using this helper. We decided, for them, that their custom translations are safe and don't need escaping. This might burn us a bit with the security team.
- There might be some use case where someone wants to actual escape all the HTML tags in a custom translation. 

What would you think about having consumers of `runtimeTranslation` explicitly choose whether or not they call it with triple or double braces? The `translate` method in Jambo could contain an `shouldUnescape` boolean or something like that:

`{ translate phrase='<div>Blah [[value]]</div>' value=profile.name shouldUnescape=true }`

that would be preprocessed into:

`{{{ runtimeTranslation phrase='<div>Blah [[value]] translated</div>' value=profile.name }}}`

This puts the responsibility on the invoker of the `translate` helper to choose if the phrase should be unescaped or not. When they add custom translations, they must decide explicitly if they should be escaped. We're not on the hook for that decision (and it's consequences).

Reply:
I really like that idea! I think it makes a lot of sense. Combining that with your other comment, intermixed markdown then would only be supported with the HBS helper with shouldUnescape=true.